src/main/java/dev/openfeature/sdk/AbstractStructure.java

```diff
@@ -32,4 +37,11 @@ public Map<String, Object> asObjectMap() {
                         (accumulated, entry) -> accumulated.put(entry.getKey(), convertValue(entry.getValue())),
                         HashMap::putAll);
     }
+
+    protected Map<String, Value> getAttributes() {
+        if (attributes == null) {
+            attributes = new HashMap<>();
+        }
+        return attributes;
+    }
```

Comment:
perf: lazily initialize.

Reply:
Makes it non thread safe?

Reply:
Excellent point. I've reverted this because it's not a even a substantial savings. The bench result barely changes.